Modules/Synchronization/src/Synchronization.Application/DomainEvents/Incoming/PeerDeleted/PeerDeletedDomainEventHandler.cs

```diff
@@ -25,16 +26,13 @@ public async Task Handle(PeerDeletedDomainEvent domainEvent)
 
     private async Task CreateExternalEvent(PeerDeletedDomainEvent @event)
     {
-#pragma warning disable IDE0037
-        var payload = new
-        {
-            RelationshipId = @event.RelationshipId,
-            DeletionDate = SystemTime.UtcNow
-        };
-#pragma warning restore IDE0037
         try
         {
-            await _dbContext.CreateExternalEvent(@event.PeerOfDeletedIdentity, ExternalEventType.PeerDeleted, payload);
+            var payload = new PeerDeletedExternalEvent.PayloadT { RelationshipId = @event.RelationshipId, DeletionDate = SystemTime.UtcNow };
```

Comment:
I would name the `PayloadT` class either `Payload` or `EventPayload` or has the `T` suffix a special meaning?

Reply:
The reason for the T was that `Payload` is an inner class of a class that has a `Payload` property. And you cannot have a class and a property with the same name in the same class.

But I like `EventPayload`. Will rename it.